tests/security/stig/oscap_xccdf_eval_remote.pm

```diff
@@ -0,0 +1,47 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Test "stig" hardening in the "scap-security-guide": detection mode with remote
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#93886, poo#104943
+
+use base 'stigtest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use bootloader_setup qw(add_grub_cmdline_settings);
+use power_action_utils "power_action";
+
+sub run {
+    my ($self) = @_;
+
+    add_grub_cmdline_settings('ignore_loglevel', update_grub => 1);
+    power_action('reboot', textmode => 1);
+    $self->wait_boot(textmode => 1);
+
+    select_console 'root-console';
+
+    # Get ds file and profile ID, ds file
```

Comment:
Suggest changes:
Get ds file and profile ID of ds file

Reply:
deleted ", ds file" (it is typo)